spk/fuse-exfat/src/installer.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# Package
+PACKAGE="exfat"
+DNAME="exfat"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put fuse-exfat & exfat-utils in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/sbin/mount.exfat-fuse /usr/sbin/mount.exfat
```

Comment:
Links to /usr/sbin/ required for automounting according to next article 
http://stackoverflow.com/questions/1554178/how-to-register-fuse-filesystem-type-with-mount8-and-fstab